src/libsync/owncloudpropagator.h

```diff
@@ -227,7 +227,7 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
 private slots:
     bool possiblyRunNextJob(PropagatorJob *next) {
         if (next->_state == NotYetStarted) {
-            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)), Qt::QueuedConnection);
+            connect(next, SIGNAL(finished(SyncFileItem::Status)), this, SLOT(slotSubJobFinished(SyncFileItem::Status)));
```

Comment:
Please don't commit this, we established it is wrong.  
Or do you fix it later?